paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -61,10 +67,18 @@ internal fun CustomerSheetScreen(
                         )
                     }
                     is CustomerSheetViewState.AddPaymentMethod -> {
-                        AddCard(
-                            viewState = viewState,
-                            viewActionHandler = viewActionHandler,
-                        )
+                        if (CustomerSheetACHV2Flag) {
```

Comment:
Instead of this, can you adopt the new `FeatureFlags` system here?

Feature flags are enabled in debug builds and disabled in release builds, and you can easily override it in tests with `FeatureFlagTestRule`.